auto_py_to_exe/web/js/i18n.js

```diff
@@ -303,6 +334,7 @@ const translationMap = {
                 de: 'ARGUMENTE',
                 pt_br: 'ARGUMENTOS',
                 sp: 'ARGUMENTOS',
+                tr: 'DİZİN',

```

Comment:
I feel like this one might be a dupe of the one above too?

Reply:
Yes, you are right, interestingly it is shifted, sorry :)